### PR TITLE
Payment service

### PR DESCRIPTION
**Pull request by Lagrida**
Payment service

### CODE REVIEW

File: source/user/index.php [hunk 1]
```diff
@@ -1,4 +1,7 @@
 <?php
+/*
+ * Payment Service
```
Comment (wizzybenson) [inline source/user/index.php:3]: Hi @Lagrida , is this comment about payment supposed to be here in user source?

Comment (Lagrida) [reply source/user/index.php:3]: sorry, it's not intentional error

Comment (wizzybenson) [reply source/user/index.php:3]: No worries, please remove and recommit. Also give more meaningful commit messages.
something like "removed unintentional comment from user source"

Comment (Lagrida) [reply source/user/index.php:3]: okey thank you, i will pull again

Comment (wizzybenson) [reply source/user/index.php:3]: Not pull, I think just make your changes and commit then push. There is a conflict here we would fix tomorrow.

Comment (Lagrida) [reply source/user/index.php:3]: okey then

**Pull request closed without merging**